src/forward/lib.jl

```diff
@@ -62,5 +65,15 @@ _pushforward(dargs, ::typeof(getproperty), x, f) =
 @tangent literal_getproperty(t, ::Val{i}) where i =
   getproperty(t, i), (ṫ, _) -> getproperty(ṫ, i)
 
+@tangent literal_getproperty(t::DataType, ::Val{i}) where i =
+  getproperty(t, i), (ṫ, _) -> getproperty(t, i)
+
 @tangent literal_getindex(t, ::Val{i}) where i =
   getindex(t, i), (ṫ, _) -> getindex(ṫ, i)
+
+import Base.Broadcast.broadcasted
+@tangent function broadcasted(f, args...)
+  broadcasted(f, args...), (ḟ, dargs...) -> begin
+    broadcasted(f, dargs...)
```

Comment:
This doesn't seem correct at the moment, e.g. for `a .* b`.

Reply:
Yeah, sorry I will fix this one